bazaarvoice/emodb
# Remove from *-api client modules

## Github Issue #

None

## What Are We Doing Here?
Currently the emodb client libraries are extremely hard to use because they wrap in outdated version of many common libraries, such as guava, that lead to dependency conflicts for the user. This PR is the first step of many to remove guava from our client libraries.

This PR removes guava only from client modules that directly depend on guava. If they wrap in guava indirectly, such as through Dropwizard, then they are not addressed here and will be addressed later.

## How to Test and Verify

1. Check out this PR
2. Review the changes from each commit individually
3. For each commit, run `mvn dependency:tree -Dverbose` inside of each affected module and confirm that it no longer contains guava in its dependency tree.

## Risk

### Level 

`Medium`

### Required Testing

`Regression` and `Manual`

### Risk Summary

There are no intended functionality changes from this PR, so the only place for error is in edge cases introduced in logic meant to replicate guava conveniences.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
